pandas/core/window/indexers.py

```diff
@@ -265,7 +265,8 @@ def __init__(
         index_array: Optional[np.ndarray],
         window_size: int,
         groupby_indicies: Dict,
-        rolling_indexer: Union[Type[FixedWindowIndexer], Type[VariableWindowIndexer]],
+        rolling_indexer: Type[BaseIndexer],
+        indexer_kwargs: Optional[Dict],
```

Comment:
why do you need this? (`indexer_kwargs`)

Reply:
In `GroupbyRollingIndex` I am instantiating th user's custom indexer as I know the `index_array` from each group,

so I need to do:

`UsersIndexer(index_array, window_size, **indexer_kwargs)`

Reply:
kk